include/fmt/format.h

```diff
@@ -3395,6 +3395,43 @@ inline typename buffer_context<Char>::type::iterator format_to(
                               basic_format_args<context>(as));
 }
 
+namespace internal {
+
+template<typename... Ts>
+struct void_ { typedef void type; };
+
+template <typename T, typename Enable = void_<>::type>
+struct it_category { typedef int type; };
```

Comment:
Do I understand correctly that `int` is arbitrary here and anything that is not `std::output_iterator_tag` would do? Why not use `void`? =)

Reply:
Well, `void` cannot be used as an argument type to a function call. I've settled on `std:.false_type`now.